_extras/guide.md

```diff
@@ -36,7 +36,7 @@ ____
   - Run openrefine.exe
 
 * If a learner is unable to install OpenRefine on their computer due to IT restrictions for example, there are cloud services available that they could try:
-  - [openrefineder](https://github.com/betatim/openrefineder/) using MyBinder [![Binder](https://mybinder.org/badge.svg)](https://mybinder.org/v2/gh/betatim/openrefineder/6ba108b?urlpath=%2Fopenrefine) (OpenRefine 3.4.1, free to use without registration, [restricted](https://mybinder.readthedocs.io/en/latest/faq.html#how-much-memory-am-i-given-when-using-binder) to 1-2 GB RAM and server will be deleted after 10 minutes of inactivity)
+  - [openrefineder](https://github.com/betatim/openrefineder/) using MyBinder [![Binder](https://mybinder.org/badge.svg)](https://mybinder.org/v2/gh/betatim/openrefineder/6ba108b?urlpath=%2Fopenrefine) (free to use without registration, [restricted](https://mybinder.readthedocs.io/en/latest/faq.html#how-much-memory-am-i-given-when-using-binder) to 1-2 GB RAM and server will be deleted after 10 minutes of inactivity)
```

Comment:
I think it's important to note that if you use the MyBinder version it's OpenRefine 3.4.1 rather than the current one so I think I'd prefer to leave this specific version mentioned

Reply:
This might also be a useful instructor note.  Some workshops need to use the MyBinder version rather than local installs.

Reply:
OK, amended offline.